app/controllers/external_activities_controller.rb

```diff
@@ -288,6 +288,20 @@ def matedit
     end
   end
 
+  def archive
+    authorize @external_activity
+    @external_activity.archive!
+    flash[:notice]= t("matedit.archive_success", {name: @external_activity.name})
+    redirect_to :search  #TBD:  Where to go?
```

Comment:
I don't know if this was cleared up in the meeting. But I agree that going to search is the best place.